lib/helpers.js

```diff
@@ -225,6 +225,10 @@ _describe.whenCalledRemotely = function(verb, url, data, cb) {
     urlStr = '/<dynamic>';
   }
 
+  if (this.request === undefined) {
+    throw new Error('App is not specified. Please use lt.beforeEach.withApp to specify the app.');
+  }
```

Comment:
In your code, `this` points to `_describe`. However, the `request` variable is set on the `beforeEach` context provided by Mocha.

You need to move this check into `beforeEach` few lines below.